src/stacks-hierarchy/Root_PurchaseOverviewScreen/Root_PurchaseOverviewScreen.tsx

```diff
@@ -192,6 +192,7 @@ export const Root_PurchaseOverviewScreen: React.FC<Props> = ({
                 preassignedFareProduct,
                 travelDate,
                 headerLeftButton: {type: 'back'},
+                mode: params.mode,
```

Comment:
Is `mode` ever set to `'Ticket'` here, or is it undefined when navigating from the ticketing tab?